src/pages/whats-happening/index.mdx

```diff
@@ -0,0 +1,102 @@
+      href="https://ibm.box.com/s/7y7yqt1s2jrvfdeczs35rkjd8t5rb811"
+    >
+
+![Box](../../images/icon/box-logo.svg)
+
+</ResourceCard>
+  </Column>
+
+  <Column colMd={4} colLg={4} noGutterSm>
+    <ResourceCard
+      subTitle="Presentation slides"
+      actionIcon="launch"
+      href="https://www.figma.com/proto/4FYUucHe9mN2r8V19Tlno3/Carbon-for-IBM.com-v2-Presentation?page-id=86%3A62780&type=design&node-id=86-65171&viewport=1899%2C8211%2C0.32&scaling=min-zoom"
+    >
+
+![Figma](../../images/icon/figma.svg)
+
+</ResourceCard>
+  </Column>
+
+</Row>
+
+### What are the benefits?
+
+- Fully support Carbon v11 updates to color and type tokens. Also includes Carbon v11 component and accessibility updates
+- Improvements to package build which make for faster compilation time
+- Faster page load time
+- Upgrade of Web Components Lit v1 to Lit v2
+- Simplified component list with more flexibility for each component
+- Component enhancements for authoring such as content section and Masthead v2
+- Improvements to spacing logic across Layout and UI components
+- Accessibility improvements for components
+
+### Long-term Release timeline
+
+![The Long Term Support timeline](../../images/release-schedule/dotcom-lts.png)
+
+### FAQ
+
+#### What is the Carbon for IBM.com v2 beta and when will it be ready?
+
+Beta version of v2 is planned for end of June 2023. This will be the first pre-release version of the library. It will be the first opportunity for adopters/users to work with the package and provide feedback to the team regarding any bugs or differences that weren’t expected. See Long Term Support (LTS) for the timeline above.
+
+#### When are we launching Carbon for IBM.com v2 production?
+
+Production release is planned for end of September 2023. This will be when the package is considered General Availability (GA). See Long Term Support (LTS) for the timeline above.
+
+#### What is required for migration?
+
+We are working on migration guidance as we build out the v2 package. More resources and links will be provided here, so check back for more details.
+
+#### When do adopters have to migrate?
+
+Adopters should plan to migrate their application, webpages, and/or docs to Carbon for IBM.com v2 _before the end of maintenance of v1_. Ensure you have built the time for migration into your planning for 2024. If you remain on v1 after the end of maintenance, your application could have a risk with unresolved security vulnerabilities and/or outdated upstream dependencies. See Long Term Support (LTS) for the timeline above.
+
+#### What frameworks will be supported?
+
+With Carbon for IBM.com v2, the team will officially support the following frameworks:
+
+- Carbon for IBM.com Web Components
+- Carbon for IBM.com Web Components with React wrappers
+- Carbon Web Components v2
+
+#### Will Carbon for IBM.com React v1 be deprecated?
+
+Yes, all teams that rely on the Carbon for IBM.com React package can migrate to the React wrappers.
```

Comment:
Not sure if this is still necessary with the changes to the previous question. But we can leave it in if you think it provides more clarity.